pkg/kubelet/server/server.go

```diff
@@ -219,7 +219,7 @@ type PodResourcesProviders struct {
 
 // ListenAndServePodResources initializes a gRPC server to serve the PodResources service
 func ListenAndServePodResources(endpoint string, providers podresources.PodResourcesProviders) {
-	server := grpc.NewServer(podresourcesgrpc.WithRateLimiter(podresourcesgrpc.DefaultQPS, podresourcesgrpc.DefaultBurstTokens))
+	server := grpc.NewServer(apisgrpc.WithRateLimiter(apisgrpc.DefaultQPS, apisgrpc.DefaultBurstTokens, "podresource"))
```

Comment:
if we change the order of parameters (I think we should) this will need to be changed

Reply:
done